java/memory/memory-core/src/main/java/org/apache/arrow/memory/rounding/DefaultRoundingPolicy.java

```diff
@@ -44,20 +44,16 @@ public class DefaultRoundingPolicy implements RoundingPolicy {
 
   static {
     int defaultPageSize = Integer.getInteger("org.apache.memory.allocator.pageSize", 8192);
-    Throwable pageSizeFallbackCause = null;
     try {
       validateAndCalculatePageShifts(defaultPageSize);
     } catch (Throwable t) {
-      pageSizeFallbackCause = t;
       defaultPageSize = 8192;
     }
 
     int defaultMaxOrder = Integer.getInteger("org.apache.memory.allocator.maxOrder", 11);
-    Throwable maxOrderFallbackCause = null;
     try {
       validateAndCalculateChunkSize(defaultPageSize, defaultMaxOrder);
     } catch (Throwable t) {
-      maxOrderFallbackCause = t;
```

Comment:
Possibly the intent was to log them below?

Reply:
I see, how should we log this, any preferred practice?

Reply:
Hmm, we can just leave it out. Clearly nobody complained before.